chore(apache-maven): remove Nexus and links to it (#291)
camunda/camunda-docs-static: get-started/content/apache-maven.md

### Enterprise Edition

> The `private` URL is a virtual repository, which aggregates a multitude of repositories.

Reviewer: @Langleu Can we specify exactly which "private" URL we are referring to here. Can we consider mentioning either complete URL or link to a private URL?

Author: sure we can link it but it's already in the code example directly below it.

Author: linked the url

> Those include customer enterprise artifacts and all public artifacts from `public` to make builds easier if those dependencies are needed.

Reviewer: @Langleu The same as above. Can we specify exactly which "public" URL we are referring to here. Can we consider mentioning either complete URL or link to a public URL?

Author: linked the url

### Enterprise Edition
The [private](https://artifacts.camunda.com/artifactory/private/) URL is a virtual repository, which aggregates a multitude of repositories.

> Those include customer enterprise artifacts and all public artifacts from [public](https://artifacts.camunda.com/artifactory/public/) to make builds easier if those dependencies are needed.

Reviewer 2: You can talk directly to the reader.

```suggestion
Those include customer enterprise artifacts and all public artifacts from [public](https://artifacts.camunda.com/artifactory/public/) to make builds easier if you need those dependencies.
```

}
```

### Browse Camunda Artifact Storage

> In order to browse the Camunda artifacts, here are the links which can be used.

Reviewer 2: ```suggestion
In order to browse the Camunda artifacts, here are the links which you can use.
```
